internal/convert/pod.go

```diff
@@ -104,11 +108,19 @@ func toPodTemplate(serviceConfig latest.ServiceConfig, labels map[string]string,
 	return tpl, nil
 }
 
-func toImagePullPolicy(image string) apiv1.PullPolicy {
-	if strings.HasSuffix(image, ":latest") {
-		return apiv1.PullAlways
+func toImagePullPolicy(image string, specifiedPolicy string) (apiv1.PullPolicy, error) {
+	if len(specifiedPolicy) == 0 {
```

Comment:
It's a pure nit, but I prefer the notation `if specifiedPolicy == ""`. With this notation it is more obvious that specifiedPolicy is a `string` and not a `slice`.

Reply:
sorry, C++ habits here :D